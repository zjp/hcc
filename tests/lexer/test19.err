FATAL [5,1]: Unterminated string literal ignored
FATAL [6,1]: Unterminated string literal ignored
FATAL [7,1]: String literal with bad escape sequence ignored
FATAL [8,1]: Unterminated string literal  with bad escape sequence ignored
