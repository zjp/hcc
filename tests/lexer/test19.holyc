""
"&!88"
"use \n to denote a newline character"
"use \" to  for a quote and \\ for a backslash"
"unterminated
"also unterminated \"
"backslash followed by space: \ is not allowed"
"bad escaped character: \a AND not terminated
