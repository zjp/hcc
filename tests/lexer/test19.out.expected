STRINGLIT:"" [1,1]
STRINGLIT:"&!88" [2,1]
STRINGLIT:"use \n to denote a newline character" [3,1]
STRINGLIT:"use \" to  for a quote and \\ for a backslash" [4,1]
EOF [9,1]
